uk.co.compucorp.civicrm.tasksassignments/xml/Menu/tasksassignments.xml

```diff
@@ -87,4 +87,10 @@
     <title>FileZip</title>
     <access_arguments>access Tasks and Assignments</access_arguments>
   </item>
+
+  <!-- API endpoint for angular: unclosed cases -->
```

Comment:
I think this is an Ajax endpoint, not an API. But, anyway, this comment is kind of useless. What if I want to use for something not angular related?